lib/puppet/type/cisco_bgp_neighbor_af.rb

```diff
@@ -387,6 +389,24 @@ def self.title_patterns
     newvalues(:true, :false, :default)
   end
 
+  newproperty(:route_map_in) do
+    desc 'route-map in state. Valid values are a string defining the name ' \
```

Comment:
How about just munging it to_s?

   munge do |value|
     value = :default if value == 'default'
     value.to_s
   end